### ISSUE TITLE
Bug Report - AE Self Service Repair Doesn't work

### ISSUE THREAD

**Issue by JSterlacci21** [RecklessRP2/rlrp-issue-tracker#37]
### Reporter

jsterlacci

### Summary

When attempting to use the AE Self service, you add a repair to your cart, and then click complete, however it will just take your money and leave your car broken

### Steps to Reproduce

1. Break your car
2. Go to AE Self Service
3. Add a Repair to your cart

### Expected Behavior

Repairs car

### Actual Results

Keeps car same amout of broken, while taking the $80 from you each time.

**Comment by iGreatWhite** [comment]
I do not believe its self service. That is how you submit a work order for a mechanic to work on your car. I have not tried it in a bit but sure that's not how it worked in the past.
